FE/winkel-app/src/app/shopping-cart/payment/payment.service.ts

```diff
@@ -11,28 +11,32 @@ export class PaymentService {
 
   constructor(private http: HttpClient) {}
 
+  private headers = new HttpHeaders({
+    'Content-Type': 'application/json',
+  });
+
   getOrder(orderId: string): Observable<any> {
-    return this.http.get('http://localhost:3000/api/v1/order/init');
+    return this.http.get('http://localhost:3000/api/v1/order/init', { headers: this.headers });
```

Comment:
remove "http://localhost:3000":
return this.http.get('/api/v1/order/init', { headers: this.headers });